src/main/java/com/wellsky/workflow/WorkflowApplication.java

```diff
@@ -6,6 +6,8 @@
 @SpringBootApplication
 public class WorkflowApplication {
 
+    private String Password = "12345";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'Password' detected in this expression, review this potentially hard-coded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://hhh-aks.eastus.cloudapp.azure.com/coding_rules#rule_key=squid%3AS2068)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "Password" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://hhh-aks.eastus.cloudapp.azure.com/coding_rules#rule_key=squid%3AS1068)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "Password" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://hhh-aks.eastus.cloudapp.azure.com/coding_rules#rule_key=squid%3AS00116)